src/main/resources/skyblocker.mixins.json

```diff
@@ -4,6 +4,8 @@
   "plugin": "de.hysky.skyblocker.compatibility.MixinPlugin",
   "compatibilityLevel": "JAVA_21",
   "client": [
+    "SoundSystemMixin",
+    "EntityRideMixin",
```

Comment:
Like translation keys, mixins are also sorted lexicographically. This makes it easier to find and edit/remove later on.